### PR TITLE
🚀 Release v1.9.0-beta.0

### PR DESCRIPTION
**Pull request by chandankumar4**
**What this PR does / why we need it**:

Add release notes for v1.9.0-beta.0

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #
Part of: https://github.com/kubernetes-sigs/cluster-api/issues/11092

/area release

### CODE REVIEW

File: CHANGELOG/v1.9.0-beta.0.md [hunk 1]
```diff
@@ -0,0 +1,317 @@
+🚨 This is a BETA RELEASE. Use it only for testing purposes. If you find any bugs, file an [issue](https://github.com/kubernetes-sigs/cluster-api/issues/new).
+
+## 👌 Kubernetes version support
+
+- Management Cluster: v1.27.x -> v1.31.x
+- Workload Cluster: v1.25.x -> v1.31.x
```
Comment (sbueringer) [inline CHANGELOG/v1.9.0-beta.0.md:6]:
```suggestion
- Management Cluster: v1.28.x -> v1.31.x
- Workload Cluster: v1.26.x -> v1.31.x
```

**Comment by Sunnatillo** [comment]
@chandankumar4 We should add [this](https://github.com/kubernetes-sigs/cluster-api/issues/11092#issuecomment-2402397477) in release notes. 
cc @sbueringer

**Comment by sbueringer** [comment]
/approve

(I'll let you figure out the remaining details)

**Pull request merged**